src/main/deltachat/controller.ts

```diff
@@ -339,6 +347,20 @@ export default class DeltaChatController extends EventEmitter {
     return DeltaChatNode.getProviderFromEmail(email)
   }
 
+  hintUpdateIfNessesary() {
+    if (
+      this._dc &&
+      Date.now() >
+        Timespans.ONE_DAY_IN_SECONDS * DAYS_UNTIL_UPDATE_SUGGESTION * 1000 +
+          BUILD_TIMESTAMP
+    ) {
+      this._dc.addDeviceMessage(
+        `update-suggestion-${VERSION}`,
```

Comment:
The general code but also this allows us to only send one notification per version that the client might be outdated. Not sure if we want to send another notification after 180 days? ^^

Reply:
not now, if you want you can mention it in the forum post though.